### PR TITLE
Config options for refresh interval and number of columns added to config page

### LINKED ISSUE #15: Add config file to allow configuration of default number of columns

### PR DESCRIPTION
**Pull request by MarkGaze**
Fixes #15 (well, replaces the need for a config file)

### CODE REVIEW

**Comment by MarkGaze** [comment]
Should also now fix #53

File: src/services/config-service.ts [hunk 1]
```diff
@@ -31,6 +33,20 @@ export class ConfigService {
     this.saveToStorage(this.urlKey, value);
   }
 
+  /**
+   * Gets the VSTS project name
```
Comment (benmatselby) [inline src/services/config-service.ts]: Can this be updated please?

File: src/services/config-service.ts [hunk 2]
```diff
@@ -31,6 +33,20 @@ export class ConfigService {
...
+   */
+  public get NumberOfColumns(): number {
+    return +this.getFromStorage(this.columnsKey) || 2;
+  }
+
+  /**
+   * Sets the VSTS project name
```
Comment (benmatselby) [inline src/services/config-service.ts]: Can this be updated please?

File: src/services/config-service.ts [hunk 3]
```diff
@@ -45,6 +61,20 @@ export class ConfigService {
     this.saveToStorage(this.projectKey, value);
   }
 
+  /**
+   * Gets the VSTS project name
+   */
+  public get RefreshInterval(): number {
```
Comment (benmatselby) [inline src/services/config-service.ts]: Can this be updated please?

File: src/services/config-service.ts [hunk 4]
```diff
@@ -45,6 +61,20 @@ export class ConfigService {
...
+    return +this.getFromStorage(this.refreshKey) || 60000;
+  }
+
+  /**
+   * Sets the VSTS project name
```
Comment (benmatselby) [inline src/services/config-service.ts]: Can this be updated please?

**Pull request merged**